packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -61,13 +75,30 @@ export class Arc extends React.Component<IArcProps, IArcState> {
           aria-label={this._getAriaLabel()}
           role="img"
         />
-        <text textAnchor={'middle'} className={classNames.insideDonutString} y={5}>
-          {this.props.valueInsideDonut!}
-        </text>
+        <g className={this._classNames.nodeTextContainer}>
+          <text
+            textAnchor={'middle'}
+            className={this._classNames.insideDonutString}
+            y={5}
+            id={'Donut_center_text'}
```

Comment:
dont hardcode ids

Reply:
Since the wrapping utility has dependency on this Id, need to keep this Id hardcoded